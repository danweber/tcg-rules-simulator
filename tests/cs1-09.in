VERSION:2
# [Your Turn] When you would play a yellow monster, by suspending this Monster, reduce the cost by 2.
FIELD1:Mon1,CS1-09 Mon2,CS1-09 
FIELD2:Victim,DW1-Van Attacker,DW1-Van Green,DW1-Bird
HAND2:DW1-Standup
SECURITY1:CS1-02 CS1-02
SECURITY2:CS2-02 CS2-02
GAME:P2 T2 MAIN 7
DECK1:DW1-Van
DECK2:DW1-Van
DUMP:DP

ECHO: -- Attacks, Victim gets penalized
CMD:ATTACK Attacker Player
STEP:AUTOANSWER:CAPTURE
DUMP:DP
ECHO -- Stand up and attack again
CMD:PLAY DW1-Standup
#Stand up the second
STEP:TO:1
# I don't know why this one is needed, there are no more choices.
STEP:TO:0
CMD:ATTACK Attacker Player
STEP:AUTOANSWER:CAPTURE
DUMP:DP

CMD:PASS
CMD:MAIN
ECHO: -- attack again, another 800 DP lost
CMD:ATTACK Mon1 Attacker
STEP:AUTOANSWER:CAPTURE
DUMP:DP

ECHO: -- pass turn, all DP losses go away
CMD:PASS
DUMP:DP
EXIT
