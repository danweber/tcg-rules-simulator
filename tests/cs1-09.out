FIELD1:Mon1,CS1-Solidsteel,7500DP Mon2,CS1-Solidsteel,7500DP
FIELD2:Victim,DW1-Vanillamon,5000DP Attacker,DW1-Vanillamon,5000DP Green,DW1-Birder,3000DP
GAME:P2 T2 MAIN 7
ECHO: -- Attacks, Victim gets penalized
CAPTURE:Solid Steel [All Turns] When a Monster becomes suspended, 1 of your opponent's Monster gets -400 DP …
CAPTURE:Solid Steel [All Turns] When a Monster becomes suspended, 1 of your opponent's Monster gets -400 DP …
CAPTURE:Target VanillaMon 3
CAPTURE:Target VanillaMon 4
CAPTURE:Target Birder 5
CAPTURE:Target VanillaMon 3
CAPTURE:Target VanillaMon 4
CAPTURE:Target Birder 5
FIELD1:Mon1,CS1-Solidsteel,7500DP Mon2,CS1-Solidsteel,7500DP
FIELD2:Victim,DW1-Vanillamon,4200DP Attacker,DW1-Vanillamon,REST,5000DP Green,DW1-Birder,3000DP
GAME:P2 T2 MAIN 7
CAPTURE:Solid Steel [All Turns] When a Monster becomes suspended, 1 of your opponent's Monster gets -400 DP …
CAPTURE:Solid Steel [All Turns] When a Monster becomes suspended, 1 of your opponent's Monster gets -400 DP …
CAPTURE:Target VanillaMon 3
CAPTURE:Target VanillaMon 4
CAPTURE:Target Birder 5
CAPTURE:Target VanillaMon 3
CAPTURE:Target VanillaMon 4
CAPTURE:Target Birder 5
FIELD1:Mon1,CS1-Solidsteel,7500DP Mon2,CS1-Solidsteel,7500DP
FIELD2:Victim,DW1-Vanillamon,3400DP Attacker,DW1-Vanillamon,REST,5000DP Green,DW1-Birder,3000DP
GAME:P2 T2 MAIN 3
ECHO: -- attack again, another 800 DP lost
CAPTURE:Solid Steel [All Turns] When a Monster becomes suspended, 1 of your opponent's Monster gets -400 DP …
CAPTURE:Solid Steel [All Turns] When a Monster becomes suspended, 1 of your opponent's Monster gets -400 DP …
CAPTURE:Target VanillaMon 3
CAPTURE:Target VanillaMon 4
CAPTURE:Target Birder 5
CAPTURE:Target VanillaMon 3
CAPTURE:Target VanillaMon 4
CAPTURE:Target Birder 5
FIELD1:Mon1,CS1-Solidsteel,REST,7500DP Mon2,CS1-Solidsteel,7500DP
FIELD2:Victim,DW1-Vanillamon,2600DP Green,DW1-Birder,3000DP
GAME:P1 T3 MAIN 3
ECHO: -- pass turn, all DP losses go away
FIELD1:Mon1,CS1-Solidsteel,REST,7500DP Mon2,CS1-Solidsteel,7500DP
FIELD2:Victim,DW1-Vanillamon,5000DP Green,DW1-Birder,3000DP
GAME:P2 T4 HATCHING 3
